src/ContainerRegistry/ContainerRegistry/Commands/UpdateAzureContainerRegistryRepository.cs

```diff
@@ -28,19 +28,19 @@ public class UpdateAzureContainerRegistryRepository : ContainerRegistryDataPlane
 
         [Parameter(Mandatory = false, HelpMessage = "Delete enable.")]
         [ValidateNotNullOrEmpty]
-        public bool DeleteEnabled { get; set; }
+        public bool? DeleteEnabled { get; set; }
```

Comment:
No idea why this breaking change was not detected, it causes other PR's [CI ](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1709765&view=logs&jobId=e1b7b984-9f58-529f-7c5a-f15f8e35cfa6&j=e1b7b984-9f58-529f-7c5a-f15f8e35cfa6&t=e8689ae8-b216-55f7-b207-4831f84ff98a)failed. Could you have a look? @wyunchi-ms @VeryEarly

Reply:
Given that the breaking changes were not detected, and the CI ran suspeciously fast (azure-powershell - powershell-core
succeeded on 26 Apr in 17m 28s), I guess it was caused by the throtling of GitHub API, causing CI unable to get the list of changed files, so some of the steps were skipped.